gossip/src/crds_value.rs

```diff
@@ -441,6 +441,12 @@ impl NodeInstance {
 
     // Returns true if the crds-value is a duplicate instance
     // of this node, with a more recent timestamp.
+    // The older instance is considered the duplicate instance because:
+    // - If a staked node is restarted it will receive its old instance value
+    //   from gossip. Considering the new instance as the duplicate would
+    //   prevent the node from restarting.
+    // - Similarly, hotswap relies on the the old instance being considered the
+    //   duplicate instance.
```

Comment:
This is misleading and might imply that hotswap relies on NodeInstance propagation to stop the first node; which is not true and also pretty dangerous. The node operator should stop the 1st node **before** changing the identity of the 2nd node to that of the 1st node.

What I meant in the comment in the other PR was that, _even if the 1st node is stopped first_, the 2nd node might still receive the 1st node NodeInstance through gossip after identity swap, which would then require the current ordering in duplicate designation.

I think we better leave references to hot-swap out to avoid this confusion. The 1st reason is already sufficient and effectively encompass the hot-swap case as well.